src/main/java/co/rsk/bitcoinj/script/Script.java

```diff
@@ -692,7 +692,17 @@ public boolean isPayToScriptHash() {
      */
     public boolean isSentToMultiSig() {
         try {
-            return !this.getRedeemScriptParser().getMultiSigType().equals(MultiSigType.NO_MULTISIG_TYPE);
+            MultiSigType multiSigType = this.getRedeemScriptParser().getMultiSigType();
+
+            // If it's a FLYOVER type, we need to get the multi sig type from the internal redeem script
+            if (MultiSigType.FLYOVER == multiSigType) {
+                List<ScriptChunk> internalRedeemScriptChunks = this.getChunks()
+                    .subList(2, chunks.size());
+                RedeemScriptParser internalRedeemScriptParser = RedeemScriptParserFactory.get(
+                    internalRedeemScriptChunks);
+                multiSigType = internalRedeemScriptParser.getMultiSigType();
+            }
```

Comment:
no big deal, but wdyt about extracting this to a method to avoid the comment? To follow uncle bob's principles :)

```suggestion
            if (MultiSigType.FLYOVER == multiSigType) {
                multiSigType = getMultiSigTypeFromInternalRedeemScript();
            }
            
            private MultiSigType getMultiSigTypeFromInternalRedeemScript(Script redeemScript) {
                 List<ScriptChunk> internalRedeemScriptChunks = redeemScript.getChunks()
                    .subList(2, chunks.size());
                RedeemScriptParser internalRedeemScriptParser = RedeemScriptParserFactory.get(
                    internalRedeemScriptChunks);
                multiSigType = internalRedeemScriptParser.getMultiSigType();
            }
            
```

Reply:
Done